interface-definitions/service-router-advert.xml.in

```diff
@@ -249,6 +249,18 @@
                       <valueless/>
                     </properties>
                   </leafNode>
+                  <leafNode name="deprecate-prefix">
+                    <properties>
+                      <help>Deprecate the prefix on shutdown</help>
+                      <valueless/>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="decrement-lifetimes">
+                    <properties>
+                      <help>Decrement the values of the preferred and valid lifetimes for the prefix over time</help>
```

Comment:
Please update the help string to: `Lifetime is decremented by the number of seconds since the last RA - use in conjunction with a DHCPv6-PD prefix.`